### DISCUSSION TITLE
Issue regarding count of fruits on front and back side

### DISCUSSION THREAD

**Discussion by isshhh9** [larics/icuas24_competition#11] [General]
For certain plants identical (both in number and position) fruits are present on front and back. Will they be counted individually or not. For eg if two fruits are present on both front and back at apparent same positions will they be counted as two or four
![Screenshot from 2023-12-24 17-58-04](https://github.com/larics/icuas24_competition/assets/150123825/e8b66de7-28c4-4165-960d-f64c50d4bc21)
 
![Screenshot from 2023-12-24 17-58-25](https://github.com/larics/icuas24_competition/assets/150123825/71a88c50-8429-4395-bfd0-e03bd9641a68)

**Comment by fpetric** [comment]
Hi, 

there are some fruits that are visible from one side only, and there are some fruits that are visible from both sides. So if the UAV sees the fruit at the same position from both sides, it should be counted only once.

**Comment by chirAAG-sehgal** [comment]
Just for clarification can you please tell the true value for the number of fruits in this plant?

**Comment by fpetric** [comment]
I think this is the case of 4 peppers, with two being seen from one side and two from the other side. You can check the .dae file of the plant model to verify (it should contain four "halves" of fruits).
